src/input/input.directive.ts

```diff
@@ -17,3 +17,13 @@ import { Directive, HostBinding } from "@angular/core";
 export class TextInput {
 	@HostBinding("class") inputClass = "bx--text-input";
 }
+
+
+@Directive({
+	selector: "[ibmTextArea]"
+})
+export class TextAreaInput {
```

Comment:
The class could probably just be `TextArea`

Reply:
Sure, will do.